src/libPMacc/include/math/vector/Vector.hpp

```diff
@@ -188,9 +188,9 @@ struct Vector : private T_Storage<T_Type, T_dim>, protected T_Accessor, protecte
     }
 
     /**
-     * Creates a Vector where all dimensions are set to the same value
+     * Give a Vector<...> were all dimensions were set to the same value
```

Comment:
This was a merge error because I had previously fixed this but merged in the old version.